custom_components/unifiprotect/config_flow.py

```diff
@@ -162,6 +165,9 @@ async def async_step_user(
                     vol.Required(
                         CONF_PORT, default=user_input.get(CONF_PORT, DEFAULT_PORT)
                     ): int,
+                    vol.Required(
+                        CONF_VERIFY_SSL, default=user_input.get(CONF_VERIFY_SSL, True)
```

Comment:
```suggestion
                        CONF_VERIFY_SSL, default=user_input.get(CONF_VERIFY_SSL, DEFAULT_VERIFY_SSL)
```


Most people won't have a valid ssl so we probably shouldn't check it by default since users won't know what to do to get it to work

 `DEFAULT_VERIFY_SSL` to `False`